jenkins/osa_onmetal.functions.groovy

```diff
@@ -370,6 +370,18 @@ def stop_api_uptime_tests() {
     """
 }
 
+def parse_results() {
+
+    String host_ip = get_onmetal_ip()
+
+    //Pull persistent, during, api, smoke results from onmetal to ES vm
+    sh """
+    git clone https://github.com/arithx/elastic-benchmark
```

Comment:
Why are we cloning from a personal repo? I thought the idea was to either include the files into this project directly (qa-jenkins-onmetal) or to move them to the OSIC repository. We could ask Daryl what he thinks?